README.md

```diff
@@ -8,10 +8,11 @@ It is fully JSON-RPC [2.0 & 1.0 compatible](http://www.jsonrpc.org/specification
 
 ![libjson-rpc-cpp logo](https://github.com/cinemast/libjson-rpc-cpp/blob/master/dev/artwork/logo.png?raw=true)
 
-**5 good reasons for using libjson-rpc-cpp in your next RPC project**
+**6 good reasons for using libjson-rpc-cpp in your next RPC project**
```

Comment:
I'd rather keep it to the 5 key features. You could add the TCP connectors to the *other good reasons* section